src/python/pants/bin/BUILD

```diff
@@ -121,4 +121,9 @@ resources(
     sources=["native_client"],
 )
 
+resources(
+    name="sandboxer",
```

Comment:
Loosely-held naming nit: Maybe `sandbox_setup`?

Reply:
Maybe, I'm open to name bikesheds, but I would like it to be an agent noun, and `sandboxer` is easy to grep for.